azkaban-db/build.gradle

```diff
@@ -16,6 +16,19 @@
  */
 apply plugin: 'distribution'
 
+task testJar(type: Jar, dependsOn: testClasses) {
+    from sourceSets.test.output
+    classifier = 'test'
+}
+
+configurations {
```

Comment:
I think testJar task is still needed. 
This change will give it more flexibility.